src/Umbraco.Core/PropertyEditors/DataEditorAttribute.cs

```diff
@@ -111,6 +111,11 @@ public DataEditorAttribute(string alias, EditorType type, string name, string vi
         /// </summary>
         public bool HideLabel { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the editor value can be copied
+        /// </summary>
+        public bool CanCopy { get; set; }
```

Comment:
@nielslyngsoe this shouldn't have been merged since this quite a large breaking change. Please be ultra mindful of merging anything, we have to try to have no breaking changes or have a discussion about how to minimize them and if they are possible. We're going to have to re-think and make changes before release.